frontend/capstone-frontend/src/DashComponents/ProviderAvailability.jsx

```diff
@@ -92,25 +92,25 @@ function ProviderAvailability() {
 
     return(
         <div className="p-15">
-            <h3 className="text-4xl font-bold text-slate-900">Set Your Preferences</h3>
+            <h3 className="dash-title">Set Your Preferences</h3>
             
             <div className="mt-8 mx-6 grid md:grid-cols-1 lg:grid-cols-2 gap-8 ">
```

Comment:
Is it possible to have _all_ the CSS come from a class in your styles.css file - particularly anything that comes from Tailwind? including this, or any webpage headers/footers?